docs/ModelZoo.md

```diff
@@ -0,0 +1,1024 @@
+# NNCF Compressed Model Zoo
+
+Here we present the results achieved using our sample scripts, example patches to third-party repositories and NNCF configuration files.
+
+- [PyTorch](#pytorch)
+  * [Classification](#pytorch-classification)
+  * [Object Detection](#pytorch-object-detection)
+  * [Semantic Segmentation](#pytorch-semantic-segmentation)
+  * [Natural Language Processing (3rd-party training pipelines)](#pytorch-nlp-huggingface-transformers-powered-models)
+- [TensorFlow](#tensorflow)
+  * [Classification](#tensorflow-classification)
+  * [Object Detection](#tensorflow-object-detection)
+  * [Instance Segmentation](#tensorflow-instance-segmentation)
+- [ONNX](#onnx)
+
+## PyTorch
+
+### PyTorch Classification
+
+<table>
```

Comment:
Fix indents (better use 2 spaces for html)

Reply:
Done